submit-queue/whitelist.txt

```diff
@@ -1,41 +1,6 @@
-brendandburns
-thockin
-mikedanese
-a-robinson
-saad-ali
-lavalamp
-smarterclayton
-justinsb
-satnam6502
-derekwaynecarr
-dchen1107
-zmerlynn
-erictune
-eparis
-caesarxuchao
-wojtek-t
-jlowdermilk
-yifan-gu
-nikhiljindal
-markturansky
-pmorie
-yujuhong
-roberthbailey
-vishh
-deads2k
-bprashanth
-cjcullen
-liggitt
 bgrant0607
```

Comment:
Surprised me so I had to look it up. https://github.com/orgs/kubernetes/teams/kubernetes-maintainers  only bgrant0607-nocc has commit access.